indico/modules/events/editing/controllers/backend/timeline.py

```diff
@@ -50,6 +50,9 @@ class RHEditingUploadFile(UploadFileMixin, RHContributionEditableBase):
     def get_file_context(self):
         return 'event', self.event.id, 'editing', self.contrib.id, self.editable_type.name
 
+    def validate_file(self, file):
```

Comment:
I'd move this into UploadFileMixin; empty files never make any sense

Reply:
I agree, should it go straight on the `if` inside `_process`? Since the `validate_file` is overriden sometimes

Reply:
yeah